sampleapp/main.go

```diff
@@ -163,16 +164,16 @@ func initProvider() {
 	tp := sdktrace.NewTracerProvider(
 		sdktrace.WithSampler(sdktrace.AlwaysSample()),
 		sdktrace.WithResource(res),
-		sdktrace.WithSyncer(exporter),
+		sdktrace.WithSyncer(traceExporter),
```

Comment:
```suggestion
		sdktrace.WithBatcher(traceExporter),
```

The `SimpleSpanProcessor` used by `WithSyncer()` is fine for trivial examples like this, but really more directed at development use.  We've run into issues with people copying its use from examples and not realizing the significant performance impact it can have, so we now recommend using `WithBatcher()` in all examples.

Reply:
Done. Updated the code.